app/filters.py

```diff
@@ -25,6 +25,17 @@ def datetime_format(value: str) -> str:
     return formatted_date
 
 
+def current_datetime_after_given(value: str) -> bool:
```

Comment:
Can we do this in the controller instead and pass a boolean to the template like `is_past_submission_deadline`?